### PR TITLE
prompt for legal terms in vmware private-cloud create

### PR DESCRIPTION
**Pull request by ctaggart**
This is a prompt similar to #2930, but for legal terms when a private cloud is created. It is also a breaking change and will need to be communicated appropriately. The internal work item is [8493765](https://msazure.visualstudio.com/One/_workitems/edit/8493765/).

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
vmware

**Comment by yungezz** [comment]
hi @qwordy could you pls help reviewing? thanks

File: src/vmware/azext_vmware/custom.py [hunk 1]
```diff
@@ -6,6 +6,20 @@
 from knack.util import CLIError
 from azext_vmware.vendored_sdks.avs_client import AVSClient
 
+LEGAL_TERMS = '''
```
Comment (yungezz) [inline src/vmware/azext_vmware/custom.py:9]: This is a long text, showing it in terminal isn't the best user experience,  today in LiftR Rps, we will show link of terms for customer reference, waht's your thinking? cc @fengzhou-msft for awareness

**Review by qwordy — APPROVED** [review]

File: src/vmware/azext_vmware/custom.py (hunk 1 above)
Comment (ctaggart) [reply src/vmware/azext_vmware/custom.py:9]: @yungezz Do you have a link to that example which shows a link of terms? I'll ask the PM & CELA if a link will work.

**Comment by ctaggart** [comment]
I've moved this back to draft while internal discussion continues.

**Review by yungezz — APPROVED** [review]

**Comment by qwordy** [comment]
Some checks were not successful

**Comment by ctaggart** [comment]
If `--accept-eula`  is passed in, the additional terms will not be displayed and there will not be a prompt to accept them. This will match PowerShell in https://github.com/Azure/azure-powershell/issues/14016.

**Comment by ctaggart** [comment]
All tests pass now. Where can we update the version of the extension and document the breaking change?

**Comment by yungezz** [comment]
> All tests pass now. Where can we update the version of the extension and document the breaking change?

hi @qwordy could you help here? thanks

**Comment by qwordy** [comment]
@ctaggart Root folder should has files `HISTORY.rst` and `setup.py`. Set version in `setup.py`. Add release history in `HISTORY.rst`. The goal is notifying users this breaking change. How do your users get news of update of this extension? If this extension has website, blog or Teams channel, you can also update them.

File: src/vmware/azext_vmware/custom.py [hunk 2]
```diff
@@ -6,6 +6,20 @@
 from knack.util import CLIError
 from azext_vmware.vendored_sdks.avs_client import AVSClient
 
+LEGAL_TERMS = '''
+LEGAL TERMS
+
+Azure VMware Solution is an Azure Service licensed to you as part of your Azure subscription and subject to the terms and conditions of the agreement under which you obtained your Azure subscription (https://azure.microsoft.com/support/legal/). The following additional terms also apply to your use of AVS:
```
Comment (yungezz) [inline src/vmware/azext_vmware/custom.py:12]: hi @ctaggart , based on one email, someone from CELA suggest below 
```
Azure VMware Solution (AVS) is an Azure Service licensed to you as part of your Azure 
```

Could you pls help to confirm? thanks

File: src/vmware/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## 0.5.4 (2020-03-13)
+
+## 0.5.3 (2020-03-12)
+
+## 0.5.2 (2020-03-10)
+
+## 0.5.1 (2020-03-10)
+
+## 0.5.0 (2020-03-09)
```
Comment (qwordy) [inline src/vmware/CHANGELOG.md:32]: I think you can delete them because there is no content.

File: src/vmware/azext_vmware/custom.py (hunk 2 above)
Comment (ctaggart) [reply src/vmware/azext_vmware/custom.py]: Confirmed & fixed.

File: src/vmware/CHANGELOG.md (hunk 3 above)
Comment (ctaggart) [reply src/vmware/CHANGELOG.md]: I added content and removed the empty versions.

**Review by yungezz — APPROVED** [review]

**Pull request merged**